platforms/android/build_static_aar.py

```diff
@@ -224,6 +234,8 @@ def main(args):
     parser.add_argument('--android_min_sdk', default="21")
     parser.add_argument('--android_target_sdk', default="26")
```

Comment:
Maybe?
```suggestion
    parser.add_argument('--android_target_sdk', default="31")
```
It would be really good to combine these two scripts into one to reduce duplication.